src/scripts/discord/Discord.ts

```diff
@@ -0,0 +1,132 @@
+class Discord implements Saveable {
+    defaults: Record<string, any> = {
+        ID: null,
+        username: null,
+    };
+    saveKey = 'discord';
+    clientID = '$DISCORD_CLIENT_ID';
+    uri = '$DISCORD_LOGIN_URI';
+    ID: KnockoutObservable<number> = ko.observable(null);
+    username: KnockoutObservable<string> = ko.observable(null);
+    codes: Array<DiscordCode> = [
+        new DiscordPokemonCode(pokemonMap['Unown (D)'], 700, 'Alternate form of Unown'),
+        new DiscordPokemonCode(pokemonMap['Unown (I)'], 700, 'Alternate form of Unown'),
+        new DiscordPokemonCode(pokemonMap['Unown (S)'], 700, 'Alternate form of Unown'),
+        new DiscordPokemonCode(pokemonMap['Unown (C)'], 700, 'Alternate form of Unown'),
+        new DiscordPokemonCode(pokemonMap['Unown (O)'], 700, 'Alternate form of Unown'),
+        new DiscordPokemonCode(pokemonMap['Unown (R)'], 700, 'Alternate form of Unown'),
+    ];
+
+    get enabled(): boolean {
+        try {
+            return !!JSON.parse('$DISCORD_ENABLED');
+        } catch (e) {
+            return false;
+        }
+    }
+
+    constructor() {
+        // User logged in, need to get details
+        const $_GET: Record<string, any> = {};
+        location.search.substr(1).split('&').map(el => el.split('=')).forEach(el => $_GET[el[0]] = el[1]);
+        if ($_GET.code) {
+            $.ajax({
+                data: $_GET,
+                type: 'get',
+                url: 'https://discord.redsparr0w.com/pokeclicker',
```

Comment:
Ah yeah thanks, this is already a config paramater, I just forget to use it 😅 
It has a description in the `_config` file, do you think it will still need a description in this file too?

Reply:
Ah, didn't notice it in the config file. I think a description just in the config file is fine, though it could still do with a link to what you are running at that site as an example

Reply:
👍 Added a link to an example gist